aten/src/ATen/native/native_functions.yaml

```diff
@@ -6276,20 +6276,13 @@
 
 - func: qr.Q(Tensor self, bool some=True, *, Tensor(a!) Q, Tensor(b!) R) -> (Tensor(a!) Q, Tensor(b!) R)
```

Comment:
@mruberry if this is an out variant, should it be renamed `qr.out`? The codegen has quite a few special case expecting the out variants to be named this way.

Reply:
I think that the convention is to use the name of the first different argument after the dot. So most functions are called `.out` because the argument is called `out`. But here we have two outputs so they are named differently.

I am almost certain to have read this somewhere in the docs but I can no longer find it now.

Reply:
This is consistent with the current qr:

https://github.com/pytorch/pytorch/blob/7df84452423f44ebe1db40a2e3463066bf954f95/aten/src/ATen/native/native_functions.yaml#L6343

But it should be fine to change it to `qr.out`, too, so we might as well.

Reply:
ok, I renamed the linalg version to `linalg_qr.out`, but left the old one unchanged. See 2a29fe53c5